test/python/pulse/test_builder.py

```diff
@@ -0,0 +1,950 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test pulse builder context utilities."""
+
+from math import pi
+import unittest
+
+import numpy as np
+
+from qiskit import circuit
+from qiskit import compiler
+from qiskit import pulse
+from qiskit.pulse import builder
+from qiskit.pulse import macros
+from qiskit.pulse import transforms
+from qiskit.pulse.instructions import directives
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import FakeOpenPulse2Q
+from qiskit.pulse import pulse_lib, instructions
+
+# pylint: disable=invalid-name
+
+
+class TestBuilder(QiskitTestCase):
+    """Test the pulse builder context."""
+
+    def setUp(self):
+        self.backend = FakeOpenPulse2Q()
+        self.configuration = self.backend.configuration()
+        self.defaults = self.backend.defaults()
+        self.inst_map = self.defaults.instruction_schedule_map
+
+
+class TestContexts(TestBuilder):
+    """test context builder contexts."""
```

Comment:
```suggestion
    """Test context builder contexts."""

```
test context contexts is a little confusing

Reply:
Fixed in f615e6380